pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunctionTest.java

```diff
@@ -203,9 +202,11 @@ public void setUp()
         _doubleMV2Values[i][j] = 1.0;
       }
 
+      float range = 1.0f - 0.0f;
+      Random random = new Random();
       for (int j = 0; j < VECTOR_DIM_SIZE; j++) {
-        _vector1Values[i][j] = Math.abs(RandomUtils.nextFloat(0.0f, 1.0f));
-        _vector2Values[i][j] = Math.abs(RandomUtils.nextFloat(0.0f, 1.0f));
+        _vector1Values[i][j] = Math.abs(random.nextFloat() * range + 0.0f);
```

Comment:
Per the documentation, `Random.nextFloat()` will generate a value from 0.0 to 1.0
```suggestion
        _vector1Values[i][j] = random.nextFloat();
```